save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectSecurityMenu.kt

```diff
@@ -40,7 +40,7 @@ private val problemsTable: FC<TableProps<ProjectProblemDto>> = tableComponent(
                 Fragment.create {
                     td {
                         Link {
-                            to = "/${cellContext.row.original.name}"
+                            to = "/project/${cellContext.row.original.organizationName}/${cellContext.row.original.projectName}/security/problems/${cellContext.row.original.id}"
```

Comment:
How about extracting `cellContext.row.original` into a variable?

Reply:
Agree. Changed it